ibis/backends/sql/compilers/base.py

```diff
@@ -331,15 +330,13 @@ class SQLGlotCompiler(abc.ABC):
         ops.Degrees: "degrees",
         ops.DenseRank: "dense_rank",
         ops.Exp: "exp",
-        ops.First: "first",
         FirstValue: "first_value",
         ops.GroupConcat: "group_concat",
         ops.IfElse: "if",
         ops.IsInf: "isinf",
         ops.IsNan: "isnan",
         ops.JSONGetItem: "json_extract",
         ops.LPad: "lpad",
-        ops.Last: "last",
```

Comment:
Every backend that supports these ops overrides the builtin definitions. As such, it made more sense to me to drop default support, letting the backends that don't support these ops avoid adding them to the `UNSUPPORTED_OPS` set.